README.md

```diff
@@ -1,95 +1,108 @@
+*gosub_scene cslib_string lowercase eye_colour
+*set eye_colour cslib_ret
+```
 
-### Coding Conventions
+### Highest Stat
 
-Any contributions to the cslib source code should adhere to the following guidelines wherever possible.
+Very often authors need to check which of the stats is the highest one, whether to display flavour text or decide which branch to follow in the story. Writing a loop in ChoiceScript is not trivial and some authors even prefer to write many lines of `*if` statements just to work around it. But what if you could check it in a single line of code?!
 
-#### Coding
+```
+*gosub_scene cslib_number max_stat "strength" "dexterity" "constitution" "intelligence" "wisdom" "charisma"
+*goto {"flavour_text_" & cslib_ret}
 
-Below follows guidance on code style when contributing to cslib.
+--------------------------------
 
-- **Plain ChoiceScript first** 
-All code in cslib modules MUST have a standard ChoiceScript implementation. It may optionally feature a \*script variant of each routine, for performance reasons, but it **must** be guarded by the *cslib_script* config variable, and be functionally equivalent to the ChoiceScript implementation.
-- **Only \*gosub_scene dependencies**
-Public cslib functions can depend on (call) one another (for e.g. string:find uses string:substring) but they **must** use \*gosub_scene and **not** \*gosub, even if they reside within the same scene. Yes, this is horrible for performance, but it guarantees correct scoping (preventing different function's variables overwriting one another) and removes a whole category of nasty bugs for end-users.
-- **cslib_ret must be set**
-All cslib functions **must** \*set cslib_ret to a value before they \*return. If the function doesn't need to return a value, it should still set cslib_ret to an empty string: "". This will help discourage over-reliance on cslib_ret, which could lead to nasty bugs, or code that is difficult to read.
-- **config variables**
-Adding any new global config variables to cslib is invasive and increases  configuration effort. Adding one should be considered a breaking change, and only ever be done as a last resort.
--  **Use \*bug for unrecoverable errors**
-Any unrecoverable error state in a function should be handled by \*bug, and contain as much useful info as possible: file name, function name, variable values etc. It should not try to print the error message to the game screen.
-- **Tests**
-All cslib functions should have tests covering at least the basic use-cases. If the function is something that isn't easily tested (like a lot of cslib:menu), try to break it down into smaller (internal) functions that can be.
-- **Comments**
-All public cslib functions should be self-documenting, i.e. contain \*comment lines before them that briefly describe: the purpose of the function, its arguments and its return value (if any).
+*label flavour_text_strength
 
+The highest stat is "strength"! Amazing! 
+```
 
-#### Naming
+### Modules
```

Comment:
Very quick sentence here explaining how cslib is split up into modules/files? Also worth mentioning that they are permitted to rely upon one another, so you can't pick and choose?

Reply:
That's a good idea.

Reply:
I did include a "depends on" column at the table, but I can't go through each file right now to check on it. Maybe we can approve as is and raise an issue to fix the table later. 

We _can_ tell users to just include all files, that would be easier for us, but I don't know how well this will age, imagine the library keeps growing and someone wants just one or two routines from `cslib_string`, they might as well just give up.